package.json

```diff
@@ -100,5 +106,19 @@
         "# check if version corresponds to changes in native files  \n if git diff --name-only ${latestVersion} HEAD | egrep \"(android/.*)|(ios/.*)\" -q; then egrep '\\.0$' -q <<< ${version}; else test $? -eq 1; fi"
       ]
     }
-  }
+  },
+  "module": "lib/module/index.js",
```

Comment:
Maybe better for `"module"` to be near to other entry options.

Reply:
Good catch!

Reply:
Fixed in 1d0b0b8.